frontend/index.js

```diff
@@ -19,6 +19,14 @@ const addTimeZoneDifference = () => {
                                 .append(hoursAheadBehind);
 };
 
+const addCurrentTimeAtKennedySpaceCentre = () => {
+    const kennedyTime = (new Date()).toLocaleTimeString(undefined, {
+        timeZone: "America/New_York"
+    })
+    return currentKennedyTime = document.getElementById('kennedyTimeNow')
+    .append(kennedyTime)
```

Comment:
😅
Just a few nitpicky bits (I'd have assumed your linter would catch these! So that's odd.)

Some semicolons, and line indentation

Reply:
Yes, I'd have expected my linter to pick that up too. That's strange. Will have to double check it.

Reply:
Done. :)